internal/ethapi/api.go

```diff
@@ -987,6 +987,9 @@ func DoCall(ctx context.Context, b Backend, args TransactionArgs, blockNrOrHash
 	// Execute the message.
 	gp := new(core.GasPool).AddGas(math.MaxUint64)
 	result, err := core.ApplyMessage(evm, msg, gp)
+	if err != nil {
```

Comment:
It's duplicated with L1001.

We can change it to 

```go
if err != nil {
	return nil, fmt.Errorf("err: %w (supplied gas %d)", err, msg.Gas())
}
```

and also remove the duplication.

Reply:
Done.